DB/src/main/java/io/deephaven/db/v2/parquet/ParquetReaderUtil.java

```diff
@@ -16,112 +12,53 @@
-                        case INT32:
-                            toString = (data) -> Arrays.toString((int[]) data);
-                            nullValue = Integer.MIN_VALUE + 1;
-                            break;
-                        case INT64:
-                            toString = (data) -> Arrays.toString((long[]) data);
-                            nullValue = Long.MIN_VALUE + 1;
-                            break;
-                        case DOUBLE:
-                            toString = (data) -> Arrays.toString((double[]) data);
-                            nullValue = Double.MIN_VALUE + 1;
-                            break;
-                        case FLOAT:
-                            toString = (data) -> Arrays.toString((float[]) data);
-                            nullValue = Float.MIN_VALUE + 1;
-                            break;
-                        case BINARY:
-                        case FIXED_LEN_BYTE_ARRAY:
-                            nullValue = null;
-                            Map<String, String> keyValueMetaData = pm.getFileMetaData().getKeyValueMetaData();
-                            String codecName = keyValueMetaData.get(ParquetTableWriter.CODEC_NAME_PREFIX + column.getPath()[0]);
-                            String codecParams = keyValueMetaData.get(ParquetTableWriter.CODEC_ARGS_PREFIX + column.getPath()[0]);
-                            ObjectCodec<Object> codec = CodecCache.DEFAULT.getCodec(codecName, codecParams);
-                            toString = (data) -> Arrays.toString((Object[]) convertArray((Binary[]) data, codec));
-                            break;
-                        case INT96:
-                            throw new UncheckedDeephavenException("Unsupported type " + column.getPrimitiveType() + " for column " + Arrays.toString(column.getPath()));
-                    }
-                    toString = applyRepetitionLevel(rg, column, toString);
-                    Function<Object, String> finalToString = toString;
-                    readRowGroup(rg, nullValue, (data) -> System.out.println(cap(finalToString.apply(data))), Arrays.asList(column.getPath()));
-                } else {
-                    Optional<Object> result = logicalTypeAnnotation.accept(new ReadRowGroupVisitor(rg, column));
-                    if (result.isPresent() && result.get() instanceof Exception) {
-                        throw new UncheckedDeephavenException("Unable to read column " + Arrays.toString(column.getPath()), (Throwable) result.get());
-                    }
-                }
-                System.out.println((System.nanoTime() - start) * 1.0 / 1000000000);
-            }
-        }
-    }
-
     @FunctionalInterface
     public interface ColumnDefinitionConsumer {
         // objectWidth == -1 means not present.
-        void accept(String name, Class<?> dataType, Class<?> componentType, boolean isGroupingColumn, String codecName, String codecArgs);
+        void accept(
```

Comment:
Too many args to not have a JavaDoc. Possibly even time to have the callback take a struct, which would be pretty convenient since you already have one (called `ColDefHelper`). As noted elsewhere, please also get rid of "db".